spec/system/help_page_spec.rb

```diff
@@ -25,4 +25,24 @@
       expect(page).not_to have_link "Help"
     end
   end
+
+  scenario "renders SDG help page link when feature is enabled" do
+    Setting["feature.help_page"] = true
+
+    Setting["feature.sdg"] = true
+    visit root_path
+    within("#navigation_bar") do
+      click_link "Help"
+    end
+
+    expect(page).to have_link "Sustainable Development Goals help", href: sdg_help_path
+
+    Setting["feature.sdg"] = nil
```

Comment:
We'we recently had many strange test failures on github actions which might (or might not :sweat_smile:) be associated to accessing the database in the middle of a browser test. How about splitting this test in two in order to avoid this issue?